### ISSUE TITLE
Attendance Members only, excluding Owners

### ISSUE THREAD

**Issue by Silentrunnerx** [OfficeDev/microsoft-teams-apps-attendance#54]
Is there a way that when the user list for attendance is pulled it only selects the Members and not members and owners?

**Comment by ricardoteixxeira** [comment]
Yes, you can add the owner in Azure Active Directory and exclude him from Members, it works for me ;)
